MediaManager/Platforms/Android/MediaManagerImplementation.cs

```diff
@@ -162,9 +162,7 @@ public override Task Play()
 
         public override async Task<IMediaItem> Play(string uri)
         {
-            var mediaItem = await MediaExtractor.CreateMediaItem(uri);
-            MediaQueue.Clear();
-            MediaQueue.Add(mediaItem);
+            var mediaItem = await base.Play(uri);
```

Comment:
I'm thinking it is not clear that base need to be called and then use that in the same method for others. Especially when you would override the method it would not be clear. What if we introduce a new method that does the clearing of the queue and adds new items to it? Then that one could return the media file to play and that can be passed to the native player.